src/filters.jl

```diff
@@ -2,7 +2,8 @@
     Filter
 
 A wrapper around a function that takes a log `Record` and returns
-a bool whether to skip logging it.
+a bool indicating whether to log it.
+`false` means "skip logging this record". 
```

Comment:
Might want to reference that we're trying to be consistent with `Base.filter` which says.

```
removing elements for which f is false
```

```suggestion
`false` means "skip logging this record", similar to `Base.filter`. 
```